docs/eventing/samples/kafka/channel/README.md

```diff
@@ -0,0 +1,147 @@
+# Apache Kafka CRD default channel
+
+You can install and configure the Apache Kafka CRD (`KafkaChannel`) as the default channel configuration in Knative Eventing. 
+
+## Prerequisites
+
+You must ensure that you meet the [prerequisites listed in the Apache Kafka overview](../README.md).
```

Comment:
@RichieEscarez  the referenced doc there does actually already link to install instructions for Strimzi, on k8s

Reply:
Thanks! Ive added comments in that file.